### PR TITLE
Writer/Editor pass on code comments for wv2 Printing esp. WinRT

### PR DESCRIPTION
**Pull request by mikehoffms**
This PR doesn't merge into "main"; it requests merging into PR https://github.com/MicrosoftDocs/edge-developer/pull/2451 where the focus is on adding WinRT code listings to the WebView2 Printing article.
AB#43413501

### CODE REVIEW

File: microsoft-edge/webview2/how-to/print.md [hunk 1]
```diff
@@ -203,14 +203,14 @@ async void PrintToDefaultPrinter(object sender, RoutedEventArgs e)
...
   {
-    // Passing null for `PrintSettings` results in default print settings used.
-    // Prints current web page with the default page and printer settings.
+    // Prints the current webpage by using the default printer settings and page settings.
+    // Passing null for PrintSettings causes the default print settings to be used.
     CoreWebView2PrintStatus printStatus = await WebView2.CoreWebView2.PrintAsync(null);
 
     if (printStatus == CoreWebView2PrintStatus.Succeeded)
     {
       dialog = new MessageDialog("Printing " + title +
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/print.md:212]: @monica-ch - fyi this revision changes a line of code, but merely a displayed string.

File: microsoft-edge/webview2/how-to/print.md [hunk 2]
```diff
@@ -349,7 +348,7 @@ CoreWebView2PrintSettings GetSelectedPrinterPrintSettings(string printerName)
 
   return printSettings;
 
-  // or
+  // or, you can do the following:
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/print.md:351]: @monica-ch - is this the intent of the "// or" line?

File: microsoft-edge/webview2/how-to/print.md [hunk 3]
```diff
@@ -671,7 +670,7 @@ async void PrintToPdf(bool enableLandscape)
   }
 
   var savePicker = new Windows.Storage.Pickers.FileSavePicker();
-  savePicker.FileTypeChoices.Add("Pdf Files", new List<string>() { ".pdf" });
+  savePicker.FileTypeChoices.Add("PDF Files", new List<string>() { ".pdf" });
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/print.md:673]: @monica-ch - tech review this Suggested Change, it's in code

File: microsoft-edge/webview2/how-to/print.md [hunk 4]
```diff
@@ -798,23 +796,23 @@ async void PrintToPdfStream()
...
     string title = webView.CoreWebView2.DocumentTitle;
 
-    // Passing null for `PrintSettings` results in default print settings used.
+    // Passing null for PrintSettings causes the default print settings to be used.
     System.IO.Stream stream = await webView.CoreWebView2.PrintToPdfStreamAsync(null);
     DisplayPdfDataInPrintDialog(stream);
 
     MessageBox.Show(this, "Printing " + title + " document to PDF Stream " +
-                ((stream != null) ? "succedded" : "failed"), "Print To PDF Stream");
+                ((stream != null) ? "succeeded" : "failed"), "Print To PDF Stream");
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/print.md:804]: @monica-ch - tech review this change in code, a typo fix from succedded to succeeded

File: microsoft-edge/webview2/how-to/print.md [hunk 5]
```diff
@@ -883,7 +881,7 @@ bool AppWindow::PrintToPdfStream()
                 DisplayPdfDataInPrintDialog(pdfData);
                 std::wstring message = L"Printing " + std::wstring(title.get()) +
                                        L" document to PDF Stream " +
-                                       ((errorCode == S_OK && pdfData != nullptr) ? L"succedded" : L"failed");
+                                       ((errorCode == S_OK && pdfData != nullptr) ? L"succeeded" : L"failed");
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/print.md:884]: instance 3
@monica-ch - tech review this change in code from succedded to succeeded

File: microsoft-edge/webview2/how-to/print.md [hunk 6]
```diff
@@ -307,7 +306,7 @@ async void PrintToPrinter()
 
     if (printStatus == CoreWebView2PrintStatus.Succeeded)
     {
-      MessageBox.Show(this, "Printing " + title + " document to printer is succeeded", "Print to printer");
+      MessageBox.Show(this, "Printing " + title + " document to printer succeeded", "Print to printer");
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/print.md:309]: @monica-ch -- innocuous change of a display string in code.  Alternatives:

document to printer succeeded.
document to printer was successful.

File: microsoft-edge/webview2/how-to/print.md [hunk 7]
```diff
@@ -328,18 +327,18 @@ async void PrintToPrinter()
   }
 }
 
-// Function to get printer name by displaying installed printers list to the user and
-// return user selected printer.
+// Gets the printer name by displaying the list of installed printers to the user and
+// return the name of the user's selected printer.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/print.md:331]:
```suggestion
// returns the name of the user's selected printer.
```

File: microsoft-edge/webview2/how-to/print.md [hunk 8]
```diff
@@ -402,19 +401,19 @@ async void PrintToPrinter(object sender, RoutedEventArgs e)
...
 string GetPrinterName()
 {
-  // Use DeviceInformation.FindAllAsync to get the list of local printers with AQS as
-  // "System.Devices.HardwareIds:~~"PRINTENUM\LocalPrintQueue"".
-  // Display the printer list to the user and get the desired printer to print.
-  // Return the user selected printer name.
+  // Use DeviceInformation.FindAllAsync to get the list of local printers with AQS as:
+  // System.Devices.HardwareIds:~~"PRINTENUM\LocalPrintQueue"
+  // Display the list of printers to the user, and get the desired printer to use.
+  // Returns the name of the user's selected printer.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/print.md:411]:
```suggestion
  // Return the name of the user's selected printer.
```

File: microsoft-edge/webview2/how-to/print.md [hunk 9]
```diff
@@ -69,12 +69,12 @@ void ShowPrintUI(object target, ExecutedRoutedEventArgs e)
...
   if (printDialog == "Browser")
   {
-    // Opens the browser print preview dialog.
+    // Opens the browser's Print Preview dialog.
     webView.CoreWebView2.ShowPrintUI();
   }
   else
   {
-    // Opens the system print dialog.
+    // Opens the system's Print dialog.
```
Comment (monica-ch) [inline microsoft-edge/webview2/how-to/print.md:77]:
```suggestion
    // Opens a system's Print dialog.
```

File: microsoft-edge/webview2/how-to/print.md (hunk 2 above)
Comment (monica-ch) [reply microsoft-edge/webview2/how-to/print.md:351]: For "or"

File: microsoft-edge/webview2/how-to/print.md (hunk 7 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/print.md:331]: fixed locally

File: microsoft-edge/webview2/how-to/print.md (hunk 3 above)
Comment (monica-ch) [reply microsoft-edge/webview2/how-to/print.md:673]: This make sense, I'll update in my PR.

File: microsoft-edge/webview2/how-to/print.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/print.md:351]: reverted 3 lines by Find: // or

File: microsoft-edge/webview2/how-to/print.md (hunk 8 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/print.md:411]: fixed locally

File: microsoft-edge/webview2/how-to/print.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/print.md:673]: using "PDF" per conversation

File: microsoft-edge/webview2/how-to/print.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/print.md:804]: the fix in displayed string is ok

**Pull request merged**